validator/webui/index.html

```diff
@@ -180,6 +176,11 @@
        are kept as an array of validation errors, and selection changes will
        fire an event ('error-selected') with the appropriate error. -->
   <dom-module id="error-list">
+    <style scope="error-list">
+      paper-listbox {
+        height: 200px; overflow-y: scroll;
```

Comment:
Nit: please make each style element it's own line.

paper-listbox {
  height: 200px;
  overflow-y: scroll;
}